### ISSUE TITLE
Community page refinements

### ISSUE THREAD

**Issue by pchote** [Fauntleroy/openra-web#8]
Just a few small nits:
- [x] Can we change "dev team" to "development team" in the top description?
- [x] "discord" &rarr; "Community Discord" so it has a similar link surface area to IRC?
- [x] Forums aren't old fashioned 😞 (and get off my lawn). How about "Join our [Community Forum](https://forum.openra.net/) for longer form discussions about events, maps, balance, or anything else! Your OpenRA forum account can be linked with your in-game player profile to display badges and join private servers".?
- [x] Missing spaces between "FAQ—if" under "Help Build OpenRA"
- [x] "OpenRA has is involved" under "Join us in the Gaming Community"
- [x] The steam group and GameReplay pages haven't been maintained for years... this might be a good time to remove them, and then merge ModDB and Itch into the social section? Or maybe ModDB could go with the resource center as "Community Content", and references to Itch moved to the download page?

**Comment by takempf** [comment]
@pchote I've made change that address most of the above, with comment:

- Oh forums are definitely old fashioned, and there's nothing wrong with that, especially given the context. In the future it might be a good idea to advertise the idea of an "OpenRA account" (that just happens to be a forum account), so that more players verify themselves (and gain those benefits). I believe future changes to the website could really help with engagement there.
- Em dashes are generally used without any spaces—however some newspapers use them with spaces as a stylistic choice. (I have added spaces)
- I simply removed Steam & GameReplays. Is there any chance we might want to add OpenRA Academy down there?

As always, a live version of the changes can be found here https://openra.kempf.dev/community
